input/docs/handbook/commands/index.md

```diff
@@ -1,23 +1,175 @@
 * `CreateFromObservable()` - Execute the logic using an `IObservable`.
 * `Create()` - Execute a synchronous Func or Action.
+* `CreateCombined()` - Read more on combining commands [here](./combining-commands).
+
+All of these methods will parameterize the resulting `ReactiveCommand` to be the return result of the method (i.e. if your async method returns `Task<string>`, your Command will be `ReactiveCommand<TInput, string>`). This means, that subscribing to the command itself returns the results of the async method as an Observable.
+
+# Synchronous commands
+
+If your command is not CPU-intensive or I/O-bound then it probably makes sense to provide synchronous execution logic. You can do so by creating a command via `ReactiveCommand.Create`:
+
+```cs
+// Creates a command with synchronous execution logic
+// which is always available for execution.
+ReactiveCommand<Unit, Unit> command = ReactiveCommand.Create(
+    () => Console.WriteLine("A reactive command is invoked!")
+);
+```
+
+# Asynchronous commands
+
+One of the most important features of `ReactiveCommand` is its built-in facilities for orchestrating asynchronous operations, commands will block re-execution while executing. `ReactiveCommand`s are fully integrated into the Reactive Extensions framework, providing an `.IsExecuting` property (of type `IObservable<bool>`) which tells you whether the command is currently executing. This is often useful if you want to trigger activity animations or you want to prevent other commands from executing while the command is executing.
+
+It is important to know, that ReactiveCommand itself as an `IObservable` will never complete or OnError - errors that happen in the async method will instead show up on the `ThrownExceptions` property. If it is possible that your async method can throw an exception, you should subscribe to `ThrownExceptions` or the exception will be rethrown on the UI thread.
+
+```cs
+// Here we declare a ReactiveCommand, OAPH and a property.
+private readonly ObservableAsPropertyHelper<List<User>> _users;
+public ReactiveCommand<Unit, List<User>> LoadUsers { get; }
+public List<User> Users => _users.Value;
+
+// Create a command with asynchronous execution logic. The 
+// command is always available for execution, it triggers
+// LoadUsersAsync method which returns Task<List<User>>.
+LoadUsers = ReactiveCommand.CreateFromTask(LoadUsersAsync);
+
+// Update the UI with a new value when users are loaded.
+// ToProperty extension method allows us to subscribe
+// to LoadUsers Observable, update our OAPH and notify
+// the UI that the value of Users property has changed.
+LoadUsers.ObserveOn(RxApp.MainThreadScheduler)
+    .ToProperty(this, x => x.Users, out _users);
```

Comment:
Your better off using the scheduler assigned in the ToProperty(this, x=> x.Users, out _users, RxApp.MainThreadScheduler);

Also these days I tend to use the ToProperty with the return value overload. Similar just more what users expect

_users = LoadUsers.ToProperty(this, x => x.Users, RxApp.MainThreadScheduler);

Also worth noting there is a overload for performance.

_users = LoadUsers.ToProperty(this, nameof(Users), RxApp.MainThreadScheduler);